scripts/create-dev-data.js

```diff
@@ -408,61 +408,237 @@ async function createData() {
+      requestedCommitSha: '40f82a2deabe157ce109dcc2cb8675ccbc2d023f',
+    })),
+    // completed on another branch
+    Build.create({
+      branch: 'longer-branch-names-might-be-truncated',
+      completedAt: addMinutes(new Date(), -56),
+      startedAt: addMinutes(new Date(), -54),
+      createdAt: addMinutes(new Date(), -54),
+      source: 'fake-build',
+      state: 'success',
+      site: nodeSite.id,
+      user: user1.id,
+      username: user1.username,
+      token: 'fake-token',
+    }).then(build => build.update({
+      clonedCommitSha: '57ce109dcc2cb8675ccbc2d023f40f82a2deabe1',
       requestedCommitSha: '57ce109dcc2cb8675ccbc2d023f40f82a2deabe1',
     })),
   ]);
 
-  const taskType = await BuildTaskType.create({
-    name: 'test',
-    description: 'test description',
+  const taskType1 = await BuildTaskType.create({
+    name: 'OWASP ZAP Vulnerability Scan',
+    description: 'This scan identifies potential website security issues like unintended exposure of sensitive data, SQL injection opportunities, cross-site scripting (XSS) flaws, and the use of components with known vulnerabilities.',
+    metadata: {
+      foo: 'bar', // no metadata is real/used yet
+    },
+    runner: 'cf_task',
+    startsWhen: 'build',
+    url: 'https://cloud.gov/pages/documentation/build-scans/',
+  });
+  await BuildTask.create({
+    buildId: nodeSiteBuilds[0].id,
+    buildTaskTypeId: taskType1.id,
+    name: 'type',
+    artifact: null,
+    count: null,
+  });
+  await BuildTask.create({
+    buildId: nodeSiteBuilds[2].id,
+    buildTaskTypeId: taskType1.id,
+    name: 'type',
+    status: 'created', // initial value, 'processing' or 'queued' haven't been made yet
+    artifact: null,
+    message: 'Scan in progress',
+    count: null,
+  });
+  await BuildTask.create({
+    buildId: nodeSiteBuilds[4].id,
+    buildTaskTypeId: taskType1.id,
+    name: 'type',
+    status: 'error',
+    artifact: null,
+    message: 'Scan could not be completed',
+    count: null,
+  });
+  await BuildTask.create({
+    buildId: nodeSiteBuilds[5].id,
+    buildTaskTypeId: taskType1.id,
+    name: 'type',
+    status: 'success',
+    artifact: 'filename-1234.html',
+    message: 'Scan successfully completed. See artifact for details.',
+    count: 0,
+  });
+  await BuildTask.create({
+    buildId: nodeSiteBuilds[6].id,
+    buildTaskTypeId: taskType1.id,
+    name: 'type',
+    status: 'success',
+    artifact: 'filename-1234.html',
+    message: 'Scan successfully completed. See artifact for details.',
+    count: 42,
+  });
+
+  const taskType2 = await BuildTaskType.create({
+    name: 'WCAG 2.2 AA Accessibility Scan',
+    description: 'This scan detects accessibility issues and provides suggestions for remediation by inspecting focusable elements, HTML tags and attributes, images, data tables, color contrast, document structure, link and button usability, and visually hidden content against the WC3’s WCAG level 2.2 A and AA.',
```

Comment:
```suggestion
    description: 'This scan detects accessibility issues and provides suggestions for remediation by inspecting focusable elements, HTML tags and attributes, images, data tables, color contrast, document structure, link and button usability, and visually hidden content against the WC3’s WCAG 2.2 Level A and AA requirements.',
```

Reply:
Actually @sknep do you want this to be the real description also? I can add to the migrations